media-sound/zynaddsubfx/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>marcan@marcan.st</email>
+		<name>Hector Martin</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proaudio@gentoo.org</email>
+		<name>Gentoo ProAudio Project</name>
+	</maintainer>
+	<use>
+		<flag name="dssi">Enable support for DSSI Soft Synth Interface</flag>
+		<flag name="zest">Enable support for the zyn-fusion zest UI</flag>
```

Comment:
Shouldn't we call this USE flag fusion? Zest is more of an internal name I think?

Reply:
Sounds good, I'll change it